scio-core/src/main/scala/com/spotify/scio/coders/instances/GuavaCoders.scala

```diff
@@ -0,0 +1,22 @@
+package com.spotify.scio.coders.instances
+import java.io.{InputStream, OutputStream}
+
+import com.google.common.hash.{Funnel, BloomFilter => gBloomFilter}
+import com.spotify.scio.coders.Coder
+import org.apache.beam.sdk.coders.AtomicCoder
+
+trait GuavaCoders {
+
+  /**
+   * Beam Coder for [[gBloomFilter]]
+   */
+  implicit def guavaBloomFilterCoder[A](implicit fa: Funnel[A]): Coder[gBloomFilter[A]] =
+    Coder.beam(
+      new AtomicCoder[gBloomFilter[A]] {
+        override def encode(value: gBloomFilter[A], outStream: OutputStream): Unit =
+          value.writeTo(outStream)
```

Comment:
So guava `BF` already has well-defined and deterministic ser/de methods?

Reply:
Yes, Guava's BF defines it, along with Strategies which defines the implementation. It serializes the strategies as well.